cmd/ui-errors.go

```diff
@@ -0,0 +1,169 @@
+/*
+ * Minio Cloud Storage, (C) 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+var (
+	uiErrInvalidConfig = newUIErrFn(
+		"Invalid value found in the configuration file",
+		"Please ensure putting a valid value in the configuration file",
+		"",
+	)
+
+	uiErrInvalidBrowserValue = newUIErrFn(
+		"Invalid browser value",
+		"Please check the passed value",
+		"Browser can only accept `on` and `off` values. To disable web browser access, set this value to `off`",
+	)
+
+	uiErrInvalidCacheDrivesValue = newUIErrFn(
+		"Invalid cache drive value",
+		"Please check the passed value",
+		"MINIO_CACHE_DRIVES: List of mounted drives or directories delimited by `;`.",
+	)
+
+	uiErrInvalidCacheExcludesValue = newUIErrFn(
+		"Invalid cache excludes value",
+		"Please check the passed value",
+		"MINIO_CACHE_EXCLUDE: List of cache exclusion patterns delimited by `;`.",
+	)
+
+	uiErrInvalidCacheExpiryValue = newUIErrFn(
+		"Invalid cache expiry value",
+		"Please check the passed value",
+		"MINIO_CACHE_EXPIRY: Cache expiry duration in days.",
+	)
+
+	uiErrInvalidCredentials = newUIErrFn(
+		"Passed credentials are not suitable for use",
+		"Please provide correct credentials",
+		`- Access key length should be between 3 and 20 characters.
```

Comment:
There is no `-` in the sentence. it is simply `Access key should be minimum 3 characters in length`  we don't have an upper limit